.github/workflows/main.yml

```
@@ -20,21 +20,20 @@ jobs:
           git fetch origin main
           git diff origin/main..HEAD -- . ':!.github/workflows' | grep -i "bd-scheduler" && echo "Found forbidden word: bd-scheduler" && exit 1 || echo "Check passed"
 
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          package_json_file: db-scheduler-ui-frontend/package.json
+
       - name: Setup Node
         uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)